plugins/binanceExchange_ws.go

```diff
@@ -15,9 +18,12 @@ import (
 )
 
 const (
-	STREAM_TICKER_FMT = "%s@ticker"
-	STREAM_BOOK_FMT   = "%s@depth"
-	TTLTIME           = time.Second * 3 // ttl time in seconds
+	STREAM_TICKER_FMT      = "%s@ticker"
+	STREAM_BOOK_FMT        = "%s@depth"
+	STREAM_USER            = "@user"
```

Comment:
`@user` is just something we made up correct? it's not actually part of the binance spec right?

If so, can you please add a comment specifying that this is not used by binance and just a key we use to reference into the `streams` map?

(we should do the same for `LAST_CURSOR_KEY` below if that is also just a key that we made up)